keyboards/mothdotmonster/ok35/readme.md

```diff
@@ -0,0 +1,15 @@
+# OK35
+
+![OK35](https://i.redd.it/o9gc51mxlol81.jpg)
+
+A 5x14 split ortholinear keyboard.
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```